inference/server/oasst_inference_server/models/chat.py

```diff
@@ -19,6 +19,7 @@ class DbMessage(SQLModel, table=True):
     reports: list["DbReport"] = Relationship(back_populates="message")
 
     parent_id: str | None = Field(None)
+    active_sibling: bool | None = Field(None, sa_column=sa.Column(pg.JSONB))
```

Comment:
What does `active_sibling` mean exactly? Feels like an odd name

Reply:
In a conversation tree you can have messages which you can switch between using the arrows at the bottom. This variable just stores whether or not that particular message was the one which was last showed. The 'sibling' part is referring to how these messages come from the same parent message so I thought it would be a good term to use. If you have a better alternative I can change it.

Reply:
Hmm ok. This feels like it is really information about the chat, not the message. Can we not just store a single `active_message` value in the chat table, which is the final message of the currently active path down the chat tree? Then from this it is already implied what siblings are active earlier in the tree

This would also simplify the endpoint, we could merge it into `update_chat` endpoint

Reply:
Sorry only seeing this now: How are the multiple generated draft messages now referenced in the database? And was is finally stored in the chat-message tree?

Reply:
The draft messages are stored as regular messages in the messages table. The message comparisons are stored in a new table called `message_eval` which contains the chat id, user id, message id of the selected message, and the message ids it was selected over.